net-p2p/go-ipfs-bin/go-ipfs-bin-0.4.22.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit bash-completion-r1 systemd user
+
+DESCRIPTION="Main implementation of IPFS"
+HOMEPAGE="https://ipfs.io/"
+SRC_URI="amd64? ( https://dist.ipfs.io/go-ipfs/v${PV}/go-ipfs_v${PV}_linux-amd64.tar.gz )
+	x86? ( https://dist.ipfs.io/go-ipfs/v${PV}/go-ipfs_v${PV}_linux-386.tar.gz )
+	arm? ( https://dist.ipfs.io/go-ipfs/v${PV}/go-ipfs_v${PV}_linux-arm.tar.gz )
+
+	https://raw.githubusercontent.com/ipfs/go-ipfs/v${PV}/misc/completion/ipfs-completion.bash -> ${P}.bash"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~arm ~x86"
+
+RDEPEND="
+	acct-group/ipfs
+	acct-user/ipfs
+	sys-fs/fuse
```

Comment:
Needs to be slotted, probably `:0` for now (but if you want, you can test if it works with fuse-3.6 too, then `:*` for slot).

Rest looks good, sorry for not following up!

Reply:
Let's stick to `:0` for now and check compatibility with other slots later when we add the non-bin ebuild variant.